### PR TITLE
Added expanded view and started unit tests

### PR DESCRIPTION
**Pull request by slava000**
@tpminhtam you're up for review

Added styled components to babel so browser dev tools inspect shows the styled component name in the class name

Added unit tests

Added expanded view:

* In the expanded view, thumbnails will not appear over the main image.  Instead, icons indicating each image in the set will appear.  These icons will be much smaller, but will have the same functionality in that clicking on them will skip to that image in the set.   Additionally the icon for the currently selected. image will be distinguishably different from the rest. 
* In the default view, clicking on the image would open the expanded view.   In the expanded view, however, clicking on the main image will zoom the image by 2.5 times. 
* While the image is zoomed, no arrow buttons or thumbnail selection icons will be available.   Upon clicking the image in this state, the user should be returned to the normal expanded image gallery view.
* The main image will still offer right and left arrows, which will have the same function of scrolling through the image set.
* After clicking, the zoomed image will be too large to display in the space provided.   In this case, the portion of the image shown within the window should correspond to the current mouse position relative to the screen.   For example, by moving the mouse right the portion of the zoomed image shown should pan to the right.  
*Furthermore, the position of the mouse relative to the centering of the zoomed image should be proportional.  If the mouse is all the way in the bottom left corner of the expanded image gallery window, the bottom left corner of the zoomed image should be displayed.   Moving the mouse to the top right should smoothly move the zoomed image available such until the top right of the image is displayed.

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -74,6 +75,7 @@ export default class App extends React.Component {
     return (
       <>
         <GlobalStyle />
+        {JSON.stringify(this.state.overview)}
```
Comment (tpminhtam) [inline client/src/components/App.jsx]: Is it for testing purpose to see what the obj looks like?

File: client/src/components/Overview/ExpandedView.jsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    document.addEventListener("keydown", this.escFunction, false);
+    document.body.style.overflow = 'hidden';
+    document.getElementById('app').style.overflow = 'hidden';
+    window.addEventListener('resize', this.updateWindowDimensions);
+  }
+  componentWillUnmount(){
+    document.removeEventListener("keydown", this.escFunction, false);
+    document.getElementById('app').style.overflow = 'unset';
+    document.body.style.overflow = 'unset';
+    window.removeEventListener('resize', this.updateWindowDimensions);
```
Comment (tpminhtam) [inline client/src/components/Overview/ExpandedView.jsx:129]: good job at preventing memory leaks!!!

**Review by tpminhtam — APPROVED** [review]

File: client/src/components/Overview/ExpandedView.jsx (hunk 2 above)
Comment (slava000) [reply client/src/components/Overview/ExpandedView.jsx:129]: thanks!

File: client/src/components/App.jsx (hunk 1 above)
Comment (slava000) [reply client/src/components/App.jsx]: ah yes (it was a good way to make sample data), but meant to take it out. thanks!

**Pull request merged**